core/src/test/java/org/apache/iceberg/TestFileScanTaskParser.java

```diff
@@ -48,28 +40,6 @@ public void testNullArguments() {
         .hasMessage("Invalid JSON string for scan task: null");
   }
 
-  @ParameterizedTest
-  @ValueSource(booleans = {true, false})
-  public void testFileScanTaskParser(boolean caseSensitive) {
-    PartitionSpec spec = TestBase.SPEC;
-    FileScanTask fileScanTask = createFileScanTask(spec, caseSensitive);
-    String jsonStr = FileScanTaskParser.toJson(fileScanTask);
```

Comment:
Should this test be removed, or adapted to use a non-deprecated method?

Reply:
There is already a [test](https://github.com/apache/iceberg/blob/main/core/src/test/java/org/apache/iceberg/TestFileScanTaskParser.java#L75) with new [ScanTaskParser](https://github.com/apache/iceberg/blob/72b39ab91dfa04d713552e70b009c24510a1cd07/core/src/main/java/org/apache/iceberg/ScanTaskParser.java)